addons/web/controllers/main.py

```diff
@@ -1423,7 +1423,7 @@ def content_assets(self, id=None, filename=None, unique=None, extra=None, **kw):
     @http.route(['/web/partner_image',
         '/web/partner_image/<int:rec_id>',
         '/web/partner_image/<int:rec_id>/<string:field>',
-        '/web/partner_image/<int:rec_id>/<string:field>/<string:model>/'], type='http', auth="public")
+        '/web/partner_image/<int:rec_id>/<string:field>/<string:model>'], type='http', auth="public")
```

Comment:
Hello, just a heads up this route should be removed in https://github.com/odoo/odoo/pull/69819 which is going to be merged soon